src/lib/sputter/net/port.cpp

```diff
@@ -0,0 +1,108 @@
+#include "port.h"
+
+#include <cstring>
+#include <iostream>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <sys/socket.h>
+#include <unistd.h>
+
+using namespace sputter::net;
+
+UDPPort::UDPPort()
+    : socketHandle(-1) {
+}
+
+UDPPort::~UDPPort() {
+  close();
+}
+
+bool UDPPort::open(uint16_t port) {
+  socketHandle = socket(AF_INET, SOCK_DGRAM, IPPROTO_UDP);
+  if (socketHandle < 0) {
+    std::cerr << "Error: Failed to create socket\n";
+    return false;
+  }
+
+  // Store port address
+  sockaddr_in bindAddress;
+  memset(&bindAddress, 0, sizeof(bindAddress));
```

Comment:
So for a POD (plain-old-data) struct in C++, you can zero-initialize with empty brackets and skip the call to `memset()`

i.e.
`sockaddr_in bindAddress = {};`

Reply:
Same comment applies to some other `memset` calls elsewhere !

Reply:
ad224eb